docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+networks:
+  esp_wireguard_network:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.18.0.0/16
+
+services:
+  esp_wireguard_examples_demo:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    image: esp_wireguard
+    container_name: esp_wireguard_client
+    networks:
```

Comment:
I guess at least at this stage, default docker network is enough (e.g. not specifying `networks` at all, and deleting `networks` at the top)? Or is there specific need for separate docker network?

Reply:
I'm not quite sure if network set-up can be removed in this case as for each container we set specific ip address.
https://docs.docker.com/reference/compose-file/services/#ipv4_address-ipv6_address

Reply:
oh yeah, you are right, having IPs dynamically assigned complicates some tests. Lets leave it then.